examples/kafka/adobe-s3-connector/README.md

```diff
@@ -0,0 +1,154 @@
+# kafka topic backup and restore
+
+Topics messages are first purged and then restore operation is performed
+
+## Prerequisites
+
+* Kubernetes 1.9+
+* K10 installed in your cluster, let's say in namespace `<kanister-operator-namespace>` Can be installed (https://docs.kasten.io/latest/install/install.html). in our case we have used `kasten-io` namespace
+* Kanctl CLI installed (https://docs.kanister.io/tooling.html#kanctl)
+
+## Setup Kafka
+Kafka can be deployed via a helm chart https://bitnami.com/stack/kafka/helm, or via an operator like strimzi.io.
+Here deploying Kafka via Strimzi Operator
+
+```bash
+# create namespace KAFKA
+$ kubectl create namespace kafka
+
+# Deploying kafka via an operator strimzi.io.
+$ kubectl apply -f 'https://strimzi.io/install/latest?namespace=kafka' -n kafka
+
+# Provision the Apache kafka and zookeeper.
+$ kubectl create -f ./kafka-cluster.yaml -n kafka
+
+# wait for the pods to be in ready state
+$ kubectl wait kafka/my-cluster --for=condition=Ready --timeout=300s -n kafka
+
+# setup kafdrop for monitoring the kafka cluster
+$ kubectl create -f kafdrop.yaml -n kafka
+
+# by default kafdrop run on port 9000, we can view it by
+kubectl port-forward kafdrop 7000:9000 -n kafka
+```
+
+## Validate producer and consumer
+Create Producer and Consumer using kafka image provided by strimzi
+```bash
+# create a producer and push data to it
+$ kubectl -n kafka run kafka-producer -ti --image=strimzi/kafka:0.20.0-kafka-2.6.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --broker-list my-cluster-kafka-bootstrap:9092 --topic blogpost
+> event1
+> event2
+> event3
+
+# creating a consumer on a different terminal
+$ kubectl -n kafka run kafka-consumer -ti --image=strimzi/kafka:0.20.0-kafka-2.6.0 --rm=true --restart=Never -- bin/kafka-console-consumer.sh --bootstrap-server my-cluster-kafka-bootstrap:9092 --topic my-topic --from-beginning
+```
+
+**NOTE:**
+* Here we have now kafka running with the broker running on service `my-cluster-kafka-bootstrap:9092`
```

Comment:
can we please put the options in tabular format and tell users what the property means?

Reply:
changes are done.